src/app/views/query-response/QueryResponse.tsx

```diff
@@ -108,9 +111,12 @@ class QueryResponse extends Component<IQueryResponseProps, IQueryResponseState>
                 calloutProps={{ gapSpace: 0 }}
                 styles={{ root: { display: 'inline-block' } }}
               >
-                <IconButton onClick={this.toggleModal} className='share-query-btn' iconProps={{
-                  iconName: 'MiniExpandMirrored'
-                }} />
+                <IconButton
+                  onClick={this.toggleModal}
+                  className='share-query-btn'
+                  iconProps={{ iconName: 'MiniExpandMirrored'}}
+                  aria-label={'Expand response'}
```

Comment:
I have noticed that you are using aria-label and ariaLabel interchangeably. Do they both work? If so, we can stick to one (ariaLabel)

Reply:
Yeah while reading the Fluent UI docs, under some components the say that the 'ariaLabel' is obsolete, so yeah I definitely think we should stick to aria-label